db/migrate/20220923092906_add_expiration_date_to_coupons.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class AddExpirationDateToCoupons < ActiveRecord::Migration[7.0]
+  def change
+    add_column :coupons, :expiration_date, :date
+
+    LagoApi::Application.load_tasks
+    Rake::Task['coupons:fill_expiration_date'].invoke
+
+    remove_column :coupons, :expiration_duration
```

Comment:
Rails/ReversibleMigration: remove_column(without type) is not reversible. (https://rails.rubystyle.guide#reversible-migration, https://api.rubyonrails.org/classes/ActiveRecord/Migration/CommandRecorder.html)